server/channels/app/migrations.go

```diff
@@ -667,6 +669,78 @@ func (s *Server) doDeleteDmsPreferencesMigration(c request.CTX) error {
 	return nil
 }
 
+func (s *Server) doConvertIncompleteGMsMigration(c request.CTX) error {
```

Comment:
Do we have a sense of the heaviness of this migration? Something we should test, perhaps?

Reply:
I'm expecting this to be a rare case but it also depends on how admins are managing their users. I'll do a synthetic test before merge.